default.nix

```diff
@@ -42,7 +42,7 @@ buildGoApplication {
       --fish <($out/bin/gomod2nix completion fish) \
       --zsh <($out/bin/gomod2nix completion zsh)
   '' + ''
-    wrapProgram $out/bin/gomod2nix --prefix PATH : ${lib.makeBinPath [ go ]}
+    wrapProgram $out/bin/gomod2nix --prefix PATH : $(dirname $(which go))
```

Comment:
Why not just remove the wrapper script then?

Reply:
We still need `go` on `$PATH` since we call out to it, but come to think of it that could just as well be a compiler flag that so we could avoid the wrapper entirely.

Reply:
But when we run the `gomod2nix` binary, it'll find the `go` in `PATH` anyway, right?

Reply:
I think I'll do that, it's more in line with what you would expect as a user.
It's strictly less pure but more pragmatical.

Reply:
I mean the binary built with `go build` directly, works out of the box, as long as the system `PATH` contains the `go`.

Reply:
I've added a commit suffixing $PATH.